examples/chat/sqlstore.go

```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/ServiceWeaver/weaver"
 	_ "github.com/go-sql-driver/mysql"
```

Comment:
Any reason we keep this dependency here?

Reply:
For the GKE case, which uses the Cloud SQL MySQL instance.